test/evp_test.c

```diff
@@ -3408,6 +3408,7 @@ static int oneshot_digestsign_test_run(EVP_TEST *t)
         t->err = "DIGESTSIGN_LENGTH_ERROR";
         goto err;
     }
+    got_len *= 2;
```

Comment:
I would do this *after* the malloc so that the buffer is exactly the right size and no longer. This way address sanitizers should pick up any subsequent writes that accidentally go beyond the length of the buffer.

Reply:
done. Added collected the extra points.